### ISSUE TITLE
Can't use hashtag in content warning summary; can't search for hashtags in CW hidden text

### ISSUE THREAD

**Issue by stuartlangridge** [tootsuite/mastodon#2071]
It's basically not possible to have a hashtag for content-warned text. If one puts a word-with-hash-symbol into the content warning "summary" (the visible part) it's not highlighted as a link (see https://mastodon.social/@sil/3084590 for an example). But if one puts a hashtag in the content-warned text (the hidden part) then that toot does not show up in the search for that hashtag. The latter seems correct (the *point* of CWs is to hide that text away unless people want to see it, so it probably shouldn't show in searches for that either) but this makes discussion of CWed topics hard to follow or reference. It seems like a hashtag in the content-warning summary could perhaps function as a hashtag without problems?

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by skddc** [comment]
+1 for hashtags in CW summaries being linked and searchable! I was a bit surprised when I couldn't find CWed posts, as I had falsely assumed the public part of it would be treated the same as public toot content. But I also think it makes perfect sense, especially for following CWed topics, as described by @stuartlangridge already.

**Comment by Cassolotl** [comment]
This is a duplicate of #699. :)

**Comment by ashfurrow** [comment]
Awesome, let's keep the discussion going in #699 👍
